src/rngToolsUi/gen3/rstid/searcher.tsx

```diff
@@ -0,0 +1,116 @@
+import { Gen3TidSidResult, rngTools } from "~/rngTools";
+import {
+  Field,
+  FormikIdFilter,
+  FormikNumberInput,
+  ResultColumn,
+  RngToolForm,
+  RngToolSubmit,
+} from "~/components";
+import React from "react";
+import { denormalizeIdFilter, IdFilterSchema } from "~/types/id";
+import { z } from "zod";
+
+type Result = Gen3TidSidResult & { offset: number };
+
+const columns: ResultColumn<Result>[] = [
+  { title: "Advance", dataIndex: "advance" },
+  { title: "Offset", dataIndex: "offset" },
+  { title: "TID", dataIndex: "tid" },
+  { title: "SID", dataIndex: "sid" },
+  { title: "TSV", dataIndex: "tsv" },
+];
+
+const Validator = z.object({
+  offset: z.number().int().min(0),
+  initial_advances: z.number().int().min(0),
+  max_advances: z.number().int().min(0),
+  filter: IdFilterSchema,
```

Comment:
Since users will only ever filter by tid, we should probably just have a tid field.

In the validator we'll want:
```suggestion
  hit_tid: z.number().int().min(0).max(65535),
```